src/CommunityToolkit.Maui.Core/Primitives/Defaults/RatingShape.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace CommunityToolkit.Maui.Core.Primitives.Defaults;
+
+/// <summary>
+/// The type of the shape to display on the rating view
+/// </summary>
+public readonly partial struct RatingShape(string pathData)
+{
+	/// <summary>
+	/// the data path data from the svg
+	/// </summary>
+	public string PathData { get; } = pathData;
+}
+
+public partial struct RatingShape
+{
+	/// <summary>
+	/// the default star shape
+	/// </summary>
```

Comment:
Let update this to describe the shape. Right now it is just repeating the property name.